push_notifications/models.py

```diff
@@ -10,91 +10,93 @@
 
 @python_2_unicode_compatible
 class Device(models.Model):
-	name = models.CharField(max_length=255, verbose_name=_("Name"), blank=True, null=True)
-	active = models.BooleanField(verbose_name=_("Is active"), default=True,
-		help_text=_("Inactive devices will not be sent notifications"))
-	user = models.ForeignKey(SETTINGS["USER_MODEL"], blank=True, null=True)
-	date_created = models.DateTimeField(verbose_name=_("Creation date"), auto_now_add=True, null=True)
+    name = models.CharField(max_length=255, verbose_name=_("Name"), blank=True, null=True)
```

Comment:
This project uses tabs, not spaces.  Please see: [CONTRIBUTING.md](https://github.com/jleclanche/django-push-notifications/blob/master/CONTRIBUTING.md)